source/tests/unit/ni_fake_non_ivi_service_tests.cpp

```diff
@@ -22,6 +25,58 @@ namespace ni {
 namespace tests {
 namespace unit {
 
+MATCHER(CustomU16Data, "")
```

Comment:
What about using these matchers: http://google.github.io/googletest/gmock_cook_book.html#matching-containers ?

Reply:
After playing around with this and carefully reading the documentation, those matchers only work on STL-like containers. (the `ElementsAreArray()` one accepts an array as input, but still acts on an STL-like container)